src/test/java/RealItemTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+import shop.RealItem;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class RealItemTest {
+
+    @Test
+    public void testSetAndGetWeight() {
```

Comment:
no need to test getters and setters. Considering that there is no other logic in the class, try to make a test to check that `toString()` method forms the string correctly. Same for the VirtualItem class